lightningd/jsonrpc.c

```diff
@@ -10,6 +10,10 @@
 #include <common/memleak.h>
 #include <common/version.h>
 #include <common/wireaddr.h>
+#include <common/bech32.h>
```

Comment:
include files need to be sorted, see: https://travis-ci.org/ElementsProject/lightning/jobs/344500119#L633